saleor/checkout/fetch.py

```diff
@@ -227,7 +227,7 @@ def fetch_checkout_lines(
                 "variant__attributes__values",
             ]
         )
-    lines = checkout.lines.prefetch_related(*prefetched_fields)
+    lines = checkout.lines.select_related().prefetch_related(*prefetched_fields)
```

Comment:
Why it is needed?

Reply:
Can't we use `prefetched_fields` to explicitly provide a list of fields that we want to prefetch? I am not sure that fetching all relations is correct solution here. 
From django docs:


> it is possible to call select_related() with no arguments. This will follow all non-null foreign keys it can find - nullable foreign keys must be specified. This is not recommended in most cases as it is likely to make the underlying query more complex, and return more data, than is actually needed.

[docs](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#select-related)

Reply:
Good point! I'm now we should just define those select_related fields explicitly, my bad for using something that had unwanted consequences.